src/ducks/appsSuggestions/AppsSuggestion.js

```diff
@@ -0,0 +1,45 @@
+import { Document } from 'cozy-doctypes'
+import { head } from 'lodash'
+import { TRANSACTION_DOCTYPE } from 'doctypes'
+
+class AppsSuggestion extends Document {
```

Comment:
I would prefer to name it AppSuggestion since we have only one app in each document right ?